pkg/influx/parser.go

```diff
@@ -99,14 +99,18 @@ func influxPointToTimeseries(pt models.Point) ([]cortexpb.TimeSeries, error) {
 		replaceInvalidChars(&name)
 
 		tags := pt.Tags()
-		lbls := make([]cortexpb.LabelAdapter, 0, len(tags)+1) // The additional 1 for __name__.
+		lbls := make([]cortexpb.LabelAdapter, 0, len(tags)+2) // An additional one for __name__, and one for internal label.
 		lbls = append(lbls, cortexpb.LabelAdapter{
 			Name:  labels.MetricName,
 			Value: name,
 		})
+		lbls = append(lbls, cortexpb.LabelAdapter{
+			Name:  "_original_format",
```

Comment:
```suggestion
			Name:  "__original_source__",
```

I'd rename this label to `__original_source__` for a few reasons:
- An internal label in Prometheus is actually prefixed by two underscores. The suffix just makes it more regular with the others like `__name__`.
- Although we're thinking of this label as "internal" its not actually internal because its not recognized by prometheus or cortex themselves. That said we could actually upstream this change to cortex, since there will be other proxies translating from other sources. So we can make this an internal label recognized by cortex and then the translators can just import the label. No need to do this now but could be a great idea when there are other translators available.
- I prefer `source` over `format` because to me its more natural to refer to influx as a source database rather than a metric format.

Reply:
And since this is going to be an important tag, we should define the string as a constant somewhere rather than as an inline literal

Reply:
And since this is going to be an important tag, we should define the string as a constant somewhere rather than as an inline literal

Reply:
I was part of the offline discussion before the initial comment was made, but thinking about it more, I think  `__proxy_source__` would be better. `original` doesn't add much information to the label name, but `source` by itself seems too generic.